### ISSUE TITLE
adapter.find method passes incorrect schema into Sequel

### ISSUE THREAD

**Issue by tsvtitan** [balderdashy/sails-mysql#224]
Adapter.find method passes incorrect schema (list of objects) if there are different connections and same collection name.

Here is workaround:

```javascript
  
  // Build find query
   var schema = connectionObject.schema;
   var _query;
        
   // remove unnecessary schema objects
   var objects = {};
     for (var key in schema) {
        if (schema[key].connection.indexOf(connectionName)!==-1) {
            objects[key] = schema[key];
         }
     }
    schema = objects;
        
   var sequel = new Sequel(schema, sqlOptions);
```

**Comment by sailsbot** [comment]
Thanks for posting, @tsvtitan.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
